modules/help_editor/ajax/help.php

```diff
@@ -32,20 +32,30 @@
     include_once "helpfile.class.inc";
 
     if (!empty($moduleName)) {
-        $helpID = \LORIS\help_editor\HelpFile::hashToID(
-            md5($subpageName ?? $moduleName)
-        );
-    }
+        try {
+            $helpID    = \LORIS\help_editor\HelpFile::hashToID(
+                md5($subpageName ?? $moduleName)
+            );
+            $help_file = \LORIS\help_editor\HelpFile::factory($helpID);
+            $data      = $help_file->toArray();
+        } catch (\NotFound $e) {
+            // Send data with empty strings so that the content can be edited
+            $data = [
+                'content' => '',
+                'topic'   => '',
+                'updated' => '-'
```

Comment:
@AlexandraLivadas sorry just noticed, this should be empty

Reply:
@zaliqarosli any other comments??

```suggestion
                'updated' => ''
```

Reply:
this should be the last one! i can re-approve once the above suggested change is comitted